designs/webviews/webviews-general.md

```diff
@@ -0,0 +1,95 @@
+# Webviews for Rich UIs in VS Code
+
+## Objective
+
+Justify expanding the criteria for using webviews within the AWS Toolkit for VS Code in order to create richer UIs than VS Code natively allows.
+
+### Out-of-scope
+
+* Architectures or frameworks for the webviews
+  * A best-practices guide to developing webviews with the architecture/framework decided on above.
```

Comment:
Roll these two lines into "Architectures, frameworks, and best practices for developing webviews" 

I think the last part of the second line is a carryover from older design docs.